.env.local.example

```
@@ -15,7 +15,16 @@ NEXT_PUBLIC_LIVEKIT_URL=wss://livekit.3speak.tv
+CHAT_JWT_SECRET=<random-secret-min-32-chars>
+# Firebase Cloud Messaging (optional — chat works without push notifications)
+# Encode your Firebase service account JSON: base64 -w0 service-account.json
+FIREBASE_SERVICE_ACCOUNT=<base64-encoded-firebase-service-account-json>
+NEXT_PUBLIC_CHAT_DEFAULT_CHANNEL=general
+
 # available themes -> bluesky, hacker, forest, cannabis, mengao, nounish, hivebr, windows95
 # available currencies -> BRL, EUR, GBP, JPY, AUD, CAD, CHF, CNY, INR (leave empty for HBD/USD)
 # Get 3Speak API key from https://embed.3speak.tv/ for video upload functionality
-# Get Image Server API key from images.3speak.tv for fallback image uploads 
\ No newline at end of file
+# Get Image Server API key from images.3speak.tv for fallback image uploads
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)